src/containers/NonBitGoRecoveryCoin/NonBitGoRecoveryCoin.tsx

```diff
@@ -98,7 +99,7 @@ function Form() {
                     maxPriorityFeePerGas: toWei(maxPriorityFeePerGas),
                   },
                   replayProtectionOptions: {
-                    chain: bitGoEnvironment === 'prod' ? 1 : 5,
+                    chain: coin === 'ethw' ? NaN : bitGoEnvironment === 'prod' ? 1 : 5,
```

Comment:
@fperalta-simtlix why are we using `NaN` as a value here?

Reply:
Because ETHw chain id is not confirmed yet. We are preparing this in cae of an eventual PoW fork of eth. I'll add a TODO comment pointing to a Jira task to update it later when confirmed.

Reply:
done